capture/generate/qrandom.py

```diff
@@ -394,42 +399,58 @@ def preprocess_and_sample(chemdf, vardict, rxndict, edict, rdict, climits):
 
         # DO THE SAMPLING
         if config.sampler == 'wolfram':
-            prdf, prmmoldf = wolfram_sampling(edict[experimentname],
+            prdf, prmmoldf, version = wolfram_sampling(edict[experimentname],
                                               rdict,
                                               vollimits,
                                               rxndict,
                                               num_wells,
                                               climits,
                                               experiment)
+            globals.set_sampler('MathematicaUniformRandom', version)
         elif config.sampler == 'default':
-            prdf, prmmoldf = default_sampling(edict[experimentname],
+            
+            prdf, prmmoldf, version = default_sampling(edict[experimentname],
                                               rdict,
                                               vollimits,
                                               rxndict,
                                               num_wells,
                                               climits,
                                               experiment)
+            globals.set_sampler('ExpertQuasiRandom', version)
         else:
             modlog.error('Encountered unexpected sampler in devconfig: {}. Exiting.'.format(config.sampler))
             sys.exit(1)
 
         erdf = pd.concat([erdf, prdf], axis=0, ignore_index=True, sort=True)
         ermmoldf = pd.concat([ermmoldf, prmmoldf], axis=0, ignore_index=True, sort=True)
+
+        model_info_dict = {'modelname': [globals.get_sampler_uid()],'participantname':['escalate']}
+        model_info_df =  pd.DataFrame.from_dict(model_info_dict)
+        exportable_model_df = pd.concat([model_info_df]*erdf.shape[0], ignore_index=True)
```

Comment:
This seems like a really big copy. It's not a problem because `model_info_df` is small, but if it ever grows, we might want to do this a different way.

Reply:
Sure.  I am not too worried about this for now.  I don't foresee a state of V2 where we are trying to do more than 200 reactions in a single batch.  Even then it is trivial.